mindjunkies/courses/migrations/0003_rename_user_coursetoken_teacher_and_more.py

```
@@ -0,0 +1,34 @@
+            name='motivation',
+            field=models.TextField(default='', help_text='Write your motivation or plan for the course, including how you plan to organize it.'),
+        ),
+        migrations.AlterField(
+            model_name='coursetoken',
+            name='course',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='tokens', to='courses.course'),
+        ),
+        migrations.AlterField(
+            model_name='coursetoken',
+            name='status',
+            field=models.CharField(choices=[('pending', 'Pending'), ('approved', 'Approved')], default='pending', max_length=10),
```

Ruff (0.8.2)

Line too long (129 > 119)

(E501)